Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -428,8 +428,11 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
     const auto eta_h = hybrid_model->m_eta_h;
     const auto rho_floor = hybrid_model->m_n_floor * PhysConst::q_e;
     const auto resistivity_has_J_dependence = hybrid_model->m_resistivity_has_J_dependence;
+    // Adding hyper-resistivity expression - mtobin 20240217
+    const auto hyper_resistivity_has_B_dependence = hybrid_model->m_hyper_resistivity_has_B_dependence;
 
-    const bool include_hyper_resistivity_term = (eta_h > 0.0) && solve_for_Faraday;
+    // const bool include_hyper_resistivity_term = (eta_h > 0.0) && solve_for_Faraday;
+    const bool include_hyper_resistivity_term = solve_for_Faraday;
```

Comment:
With the above changes this can become:
```suggestion
    const bool include_hyper_resistivity_term = hybrid_model->m_include_hyper_resistivity_term;
```